src/board/transition.ts

```diff
@@ -297,11 +298,13 @@ function updateTransitionWithKeyboardEvent<D>(
     }
   };
 
+  const isRtl = document.documentElement.dir === "rtl";
```

Comment:
I used this approach in multiple places for my dev. convenience. Need to change it to use the util to match components implementation.

Reply:
Updating to the use the utility function but I don't see an event here that is usable. We can discuss where we want the direction detection to live. Probably one place for the entire suite.